Testing/SystemTests/tests/framework/SANSILLReductionTest.py

```diff
@@ -131,6 +131,72 @@ def runTest(self):
         SANSILLIntegration(InputWorkspace='sample', OutputWorkspace='iq')
 
 
+class ILL_D22_Multiple_Sensitivity_Test(systemtesting.MantidSystemTest):
+
+    def __init__(self):
+        super(ILL_D22_Multiple_Sensitivity_Test, self).__init__()
+        self.setUp()
+
+    def setUp(self):
+        config['default.facility'] = 'ILL'
+        config['default.instrument'] = 'D22'
+        config.appendDataSearchSubDir('ILL/D22/')
+
+    def cleanup(self):
+        mtd.clear()
+
+    def validate(self):
+        self.tolerance = 1e-5
+        self.disableChecking = ['Instrument']
+        return ['iq_multi_sens', 'ILL_SANS_D22_IQ_multi_sens.nxs']
+
+    def runTest(self):
+        # Load the mask
+        LoadNexusProcessed(Filename='D22_mask.nxs', OutputWorkspace='mask')
+        LoadNexusProcessed(Filename='D22_mask_central.nxs', OutputWorkspace='mask_central')
+        LoadNexusProcessed(Filename='D22_mask_offset.nxs', OutputWorkspace='mask_offset')
+
+        # Absorber
+        SANSILLReduction(Run='241238', ProcessAs='Absorber', OutputWorkspace='Cd')
+
+        # Beam
+        SANSILLReduction(Run='241226', ProcessAs='Beam', AbsorberInputWorkspace='Cd', OutputWorkspace='Db', FluxOutputWorkspace='fl')
+
+        # Container transmission known
+        CreateSingleValuedWorkspace(DataValue=0.94638, ErrorValue=0.0010425, OutputWorkspace='ctr')
+        AddSampleLog(Workspace='ctr', LogName='ProcessedAs', LogText='Transmission')
+
+        # Container
+        SANSILLReduction(Run='241239', ProcessAs='Container', AbsorberInputWorkspace='Cd', BeamInputWorkspace='Db',
+                         TransmissionInputWorkspace='ctr', OutputWorkspace='can')
+
+        # Sample transmission known
+        CreateSingleValuedWorkspace(DataValue=0.52163, ErrorValue=0.00090538, OutputWorkspace='str')
+        AddSampleLog(Workspace='str', LogName='ProcessedAs', LogText='Transmission')
+
+        # Reference
+        SANSILLReduction(Run='344411', ProcessAs='Sample', MaskedInputWorkspace='mask_central',
+                         AbsorberInputWorkspace='Cd', BeamInputWorkspace='Db', ContainerInputWorkspace='can',
+                         OutputWorkspace='ref1', SensitivityOutputWorkspace='sens1')
+
+        SANSILLReduction(Run='344407', ProcessAs='Sample', MaskedInputWorkspace='mask_offset',
+                         AbsorberInputWorkspace='Cd', BeamInputWorkspace='Db', ContainerInputWorkspace='can',
+                         OutputWorkspace='ref2', SensitivityOutputWorkspace='sens2')
+
+        GroupWorkspaces(InputWorkspaces=['ref1', 'ref2'], OutputWorkspace='sensitivity_input')
+        CalculateEfficiency(InputWorkspace='sensitivity_input', MergeGroup=True, OutputWorkspace='sens')
```

Comment:
I am confused about this test. sens1 and sens2 are already sensitivities at this point, they are not to be passed to calculate efficiency? Am I misreading something? Perhaps you meant ref1 and ref2?

Reply:
Ok, never mind, it's correct here, but wrong in the PR testing instructions.

Reply:
Instructions corrected